portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/mfa/MultiFactorAuthentication.testcase

```diff
@@ -825,7 +825,7 @@ definition {
 				enableIp = "true");
 		}
 
-		task ("Signout and login") {
+		task ("Sign Out and Login") {
```

Comment:
In the 1139. row you didn't write "Out" with capital letter. I think it would be better to write it with "Sign out" but either way we should aim for consistency

Reply:
Thanks, it's been solved!